src/Components/Aspire.Azure.Storage.Blobs/ConfigurationSchema.json

```diff
@@ -77,6 +77,29 @@
                           "format": "uri",
                           "description": "Gets or sets the secondary storage 'System.Uri' that can be read from for the storage account if the\naccount is enabled for RA-GRS.\n\nIf this property is set, the secondary Uri will be used for GET or HEAD requests during retries.\nIf the status of the response from the secondary Uri is a 404, then subsequent retries for\nthe request will not use the secondary Uri again, as this indicates that the resource\nmay not have propagated there yet. Otherwise, subsequent retries will alternate back and forth\nbetween primary and secondary Uri."
                         },
+                        "Request100ContinueOptions": {
```

Comment:
@eerhardt  FYI, looks like the new version of Azure storage now has a new config setting.

Reply:
cc @sebastienros

Reply:
Yep - looks like it came in with https://github.com/Azure/azure-sdk-for-net/pull/43999.